src/ocm/model.js

```diff
@@ -140,6 +180,13 @@ const displayNameForData = ({
 }) => {
   const displayName = findingTypeToDisplayName(type)
 
+  const cryptoAssetToDisplayName = (cryptoAsset) => {
+    const assetType = cryptoAsset.asset_type
+
+    // if asset type is certificate, don't show all names as they are usually more irrelevant
```

Comment:
nit: "more irrelevant" than..? 🤔 
I suppose that should read "not of interest" (or similar)?